docs/example-scenario/apps/internal-apim-ase-scenario.md

```diff
@@ -0,0 +1,193 @@
+
+This example demonstrates how to configure an API Management Service deployed into a vnet which consumes APIs hosted in an App Service Environments on a secured private VNET and assumes
+- The development team will modernize the APIs back end, which is hosted on WebApps in App Service Environments internally
+- The API Management service is maintained by Administrator/s within the Organizations
+
+## Architecture
+
+![Architecture diagram][architecture]
+
+The Web APIs will be hosted as a platform as a service (PaaS) applications on Azure with Hybrid Connectivity to an on-premises network. These APIs will be available to both external and internal developers and can make use of streamlined API connections and documentation.
+
+### Components
+
+- [Azure Virtual Network][vnet] enables Azure resources to securely communicate with each other, the internet, and on-premises networks.
+- [Azure Private DNS][dns] Azure DNS provides a reliable, secure DNS service to manage and resolve domain names in a virtual network without your needing to add a custom DNS solution.
+- [Azure API Management][apim] helps organizations publish APIs to external, partner, and internal developers to unlock the potential of their data and services.
+- [Application Gateway][appgtwy] Azure Application Gateway is a web traffic load balancer that enables you to manage traffic to your web applications.
+- Internal Load Balancer [App Service Environment][ase] is an Azure App Service feature that provides a fully isolated and dedicated environment for securely running App Service apps at high scale.
+- [Azure DevOps][devops] is a service for managing your development life cycle end-to-end &mdash; from planning and project management, to code management, and continuing to build and release.
+- [Application Insights][appinsights] is a first-party, extensible Application Performance Management (APM) service for web developers on multiple platforms.
+- [Azure Cosmos DB][cosmosdb] is Microsoft's globally distributed, multi-model database service.
+
+
+### Alternatives
+
+- In an [Azure lift and shift scenario][azure-vm-lift-shift] deployed into an Azure Virtual Network, the customer could directly address their back-end servers through private IP addresses.
+- In the on-premises scenario, the API Management instance could reach back to the internal service privately via an [Azure VPN gateway and site-to-site IPSec VPN connection][azure-vpn] or [ExpressRoute][azure-er] making this a [hybrid Azure and on-premises scenario][azure-hybrid].
```

Comment:
"In the on-premises scenario", do you mean for communication to the on-premises resources or is there a different scenario?

Reply:
This means the APIs being consumed are not in Azure (ASE) but on-premises.